ReactAndroid/src/main/jni/react/jni/CatalystInstanceImpl.cpp

```diff
@@ -226,12 +225,12 @@ void CatalystInstanceImpl::jniLoadScriptFromDeltaBundle(
     jni::alias_ref<NativeDeltaClient::jhybridobject> jDeltaClient,
     bool loadSynchronously) {
 
-  auto deltaClient = jDeltaClient->cthis()->getDeltaClient();
-  auto registry = RAMBundleRegistry::singleBundleRegistry(
-    folly::make_unique<JSDeltaBundleClientRAMBundle>(deltaClient));
+  // auto deltaClient = jDeltaClient->cthis()->getDeltaClient();
+  // auto registry = RAMBundleRegistry::singleBundleRegistry(
+  //   folly::make_unique<JSDeltaBundleClientRAMBundle>(deltaClient));
```

Comment:
Did you mean to leave these here?

Reply:
The changes were done from `0.61-stable`. This code was already removed on `master`. Once we get `master` not to crash on startup (if you're curious about the crash, I've posted a question on `android` channel on React Native contributors discord), we'll reapply the changes.